account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -1157,20 +1157,10 @@ def _get_reconcile_line(
         self,
         line,
         kind,
-        is_counterpart=False,
-        max_amount=False,
-        from_unreconcile=False,
         reconcile_auxiliary_id=False,
-        move=False,
+        **kwargs,
```

Comment:
Changing this is not good for a stable version.

Reply:
:+1: